github_release.py

```diff
@@ -321,7 +322,9 @@ def get_releases(repo_name, verbose=False):
 
 
 def get_release(repo_name, tag_name):
+    time.sleep(5)
```

Comment:
Great! I was pending to issue a PR myself, so I really appreciate this PR!

I suggest a smaller sleep, something around 0.5-1 sec.

Reply:
Thanks for the feedback and the reminder.

Reply:
Hi there. I made a quick test and, indeed, 1s seems to be enough. I've been conservative, just to make sure it's reliable.

Reply:
...and I backtrack what I just said. Sleeping for 1sec is not enough and still returns some errors. I'd leave 5 seconds.